File: Extension/Java/SecondLargest.java
import java.util.Scanner;

public class SecondLargest {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        System.out.print("Enter number 1: ");
        int number = input.nextInt();

        int largest = number;
        int secondLargest = number;

        for (int i = 2; i <= 10; i++) {
            System.out.print("Enter number " + i + ": ");
            number = input.nextInt();

            if (number > largest) {
                secondLargest = largest;
                largest = number;
            } 
	else if (number > secondLargest && number < largest) {
                secondLargest = number;
            }
        }

        System.out.println("Second largest is " + secondLargest + " and largest is " + largest);
    }
}

File: Extension/Python/secondlargest.py
largest = int(input("Enter number 1: "))
second = largest

for i in range(2, 11):
    number = int(input(f"Enter number {i}: "))

    if number > largest:
        second = largest
        largest = number
    elif number > second and number < largest:
        second = number

print("Second largest is", second, "and largest is", largest)